README.md

```diff
@@ -86,3 +86,12 @@ if ('serviceWorker' in navigator) {
     })
 }
 ```
+### Installation and testing
+
+* Clone the repo from here
+* cd to ipfs-service-worker
+* npm install
+* cd examples
```

Comment:
I think you intended to write `examples/use-from-another-page`

Reply:
I think you are correct